### DISCUSSION TITLE
VisiData doesn't update SUM() when editing .xlsx spreadsheet

### DISCUSSION THREAD

**Discussion by emilBeBri** [saulpw/visidata#1825] [Q&A] [answered]
Hi so I recently started using visidata and I'm loving it - finally I can edit spreadsheets 'the vim way'. 

Today for the first time I had to open an xlsx file, that had a simple SUM() over a range of cells in it. The xslx file was created by me in LibreOffice 7.4.5.1

But in visidata, when deleting some of the cells that are being summarized, the cell with the SUM() formular does not update the sum. 

Am I missing something? It's just a really basic sheet with some simple SUM() formulas in it. 

I can construct a small example if that will help, but I'm thinking there must be something really simple I'm overlooking.

**Comment by saulpw** [comment, marked as answer]
Hi @emilBeBri !  VisiData is a data viewer, and data explorer, and data converter, maybe a data editor, but it's not a spreadsheet.  It knows nothing about SUM or any other Excel functions.  You might want to try `sc-im` for a spreadsheet that can be used in the terminal (not sure if it supports these functions either, but it's more likely.)

**Comment by emilBeBri** [comment]
Allright, thank you ! Seems like sc-im could do what I need it to, thank you for the suggestion.
